client/src/index.js

```diff
@@ -3,5 +3,11 @@ import ReactDOM from 'react-dom';
 
 import './index.css';
 import App from './App';
+import { BrowserRouter as Router } from "react-router-dom";
 
-ReactDOM.render(<App />, document.getElementById('root'));
+ReactDOM.render(
+    <Router>
+        <App />
+    </Router>,
+    document.getElementById("route")
```

Comment:
Most of the time, we won't have to modify this line here. Just make sure that you have an element in your index.html that has an id of "route"